users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_param_helper.py

```diff
@@ -19,11 +16,15 @@ def generate_specaug_params(
     Returns:
         Dict[str, List[int]]: Pre-generated parameter values for each epoch.
     """
+
+    if schedules is None:
```

Comment:
According to the typing information above, it should not be `None`.

Reply:
Right this check is useless